### PR TITLE
Created Group Info Page

### PR DESCRIPTION
**Pull request by Deepanshu-Rohilla**
WhatsApp Video 2020-05-22 at 01 13 24

### CODE REVIEW

File: lib/screens/chatscreen/chat_components/chat_bubble.dart [hunk 1]
```diff
@@ -20,11 +19,11 @@ class _ChatBubbleState extends State<ChatBubble> {
     return Container(
       padding: EdgeInsets.only(left: 16, right: 16, top: 10, bottom: 10),
       child: Align(
-        alignment: (widget.chatMessage.type == MessageType.Receiver ? Alignment.topLeft : Alignment.topRight),
+        alignment: (widget.chatMessage.sending == false ? Alignment.topLeft : Alignment.topRight),
```
Comment (arpit-saxena) [inline lib/screens/chatscreen/chat_components/chat_bubble.dart:22]: Just do `!widget.chatMessage.sending`

File: lib/screens/chatscreen/chat_components/chat_bubble.dart [hunk 2]
```diff
@@ -20,11 +19,11 @@ class _ChatBubbleState extends State<ChatBubble> {
...
         child: Container(
           decoration: BoxDecoration(
             borderRadius: BorderRadius.circular(30),
-            color: (widget.chatMessage.type == MessageType.Receiver ? chatBubbleBackgroundColorReceiver : chatBubbleBackgroundColorSender),
+            color: (widget.chatMessage.sending == false ? chatBubbleBackgroundColorReceiver : chatBubbleBackgroundColorSender),
```
Comment (arpit-saxena) [inline lib/screens/chatscreen/chat_components/chat_bubble.dart:26]: Same here.

**Comment by Deepanshu-Rohilla** [comment]
WhatsApp Video 2020-05-22 at 22 48 02
1. Created Group Page
2. Modified the Sliver App Bar.
P.S: I have created a sample trip for using it in Group Page.

**Comment by Deepanshu-Rohilla** [comment]
WhatsApp Video 2020-05-25 at 00 29 51 (1)

1.Merged the two files in group screen folder.
2.This screen is consistent in light theme also.
3.Made a screen just for group details which opens and then from there one can open chats.

**Pull request merged**